corehq/apps/change_feed/producer.py

```diff
@@ -31,6 +31,7 @@ def send_change(self, topic, change_meta):
         message = change_meta.to_json()
         try:
             self.producer.send(topic, bytes(json.dumps(message)))
+            self.producer.flush()
```

Comment:
why do we need to block on this? is this a best practice? seems like depending on the answer to the question above, it might not matter too much